test/functional/connection/connection.ts

```diff
@@ -381,33 +387,89 @@ describe("Connection", () => {
 
     });
 
-    describe("Can change postgres default schema name", () => {
+    describe.only("Different names of the same content of the schema", () => {
+
         let connections: Connection[];
         beforeEach(async () => {
-            connections = await createTestingConnections({ 
-                enabledDrivers: ["postgres"],
-                entities: [Post],
-                schemaName: "test-schema",
-                dropSchemaOnConnection: true
-            });
+            const [[connection1], [connection2]] = await Promise.all([
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [CommentV1, GuestV1],
+                    schemaName: "test-schema",
+                    dropSchemaOnConnection: true,
+                }),
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [CommentV1, GuestV1],
+                    schemaName: "another-schema",
+                    dropSchemaOnConnection: true
+                })
+            ]);
```

Comment:
looks quite complex, maybe better to write them separately without Promise.all?:

```ts
const connection1 = await createTestingConnections({ 
                    enabledDrivers: ["postgres"],
                    entities: [CommentV1, GuestV1],
                    schemaName: "first-schema",
                    dropSchemaOnConnection: true
                });

const connection2 = await createTestingConnections({ 
                    enabledDrivers: ["postgres"],
                    entities: [CommentV1, GuestV1],
                    schemaName: "second-schema",
                    dropSchemaOnConnection: true
                });

```